src/file-handler.ts

```diff
@@ -86,9 +88,56 @@ const writeToFile = async (stream: FileSystemWritableFileStream, data: ArrayBuff
     await stream.close();
 };
 
+const loadCameraPoses = (url: string, filename: string, events: Events) => {
+    return fetch(url)
+    .then((response) => response.json())
+    .then((json) => {
```

Comment:
Don't you prefer the async/await form? 😄

Reply:
I do. And since I am a modern man I have updated the code. Thanks!